docs/guide/components/notes.md

```diff
@@ -10,13 +10,18 @@ Whilst Fractal lets you create rich standalone [documentation pages](../document
 
 ## The README.md file
 
-[Compound components](../components/#compound-components) can include a `README.md` file within their component folder:
+[Components](../components/#components) and [variants](../component/#variants) can include notes in a `README.md`,
+`<component-name>.readme.md`, or `<component-name>--<variant-name>.readme.md` file within their component folder:
```

Comment:
It's a bit weird to have it in uppercase once and lowercase on the others. Should we specify that it's case insensitive?